docs/developers-guide/api-changelog.md

```diff
@@ -33,23 +33,6 @@ title: API changelog
 - `GET /api/transform/:db-id/:schema/:transform-name`, which hasn't been used internally by Metabase for ages, has
   been removed.
 
-## Metabase 0.49.0
-- `POST /api/card` and `PUT /api/card/:id`
-
-   The `dataset` key is deprecated and will be removed in a future version, most likely 50. In its place we have added a new key: `type` which is equivalent in that it distinguishes Models from Questions. `type="model"` is equivalent to `dataset=true` and `type="question"` is equivalent to `dataset=false`.
-
-- all endpoints that return data (e.g. exports in JSON, XLSX, CSV, endpoints that end in "/query")
-
-   Starting from v49, we respond to the API calls with values formatted according to the instance localization options
-
-# Additions to the API interface
-
```

Comment:
This title was added in https://github.com/metabase/metabase/pull/41268. I think this was a mistake in the long run, as the goal of the changelog is to document breaking changes only, and non-breaking additions should not be included. In this specific case, the information may be helpful for API users because it documents a way to get around a previous breaking change.